### PR TITLE
PROD-2000 DA UI bugs -> Develop

### PR DESCRIPTION
**Pull request by mmattlin**
Solves UI bugs listed in PROD-2000, including
- Wrong banner used in review page
- Title deviates from the design
- The note “A hold will be placed on your card…” should come after the checkbox
- Text in bullet point 1 in section “IMPORTANT THINGS TO KNOW ABOUT YOUR PROJECT” does not match design

### CODE REVIEW

File: src/routes/Review/index.jsx [hunk 1]
```diff
@@ -90,10 +90,10 @@ const Review = ({
     workType?.selectedWorkType === "Website Design"
```
Comment (brooketopcoder) [inline src/routes/Review/index.jsx:90]: can we get rid of this magic string?

**Review by brooketopcoder — CHANGES_REQUESTED** [review]

File: src/routes/Review/index.jsx (hunk 1 above)
Comment (mmattlin) [reply src/routes/Review/index.jsx]: Removed

**Review by brooketopcoder — APPROVED** [review]

File: src/routes/Review/index.jsx [hunk 2]
```diff
@@ -237,9 +245,9 @@ const Review = ({
                 <PageUl>
                   <li>
                     <strong>
-                      Your Dashboard is your go-to hub for managing your work.
+                      Your Dashboard is your go-to-hub for managing your work.
```
Comment (testflyjets) [inline src/routes/Review/index.jsx:248]: Calling something your "go-to" means it's the preferred way of doing that task. A "go-to-hub" doesn't make sense in this context, so I would revert the addition of the extra `-` and leave it as "go-to hub".

Comment (mmattlin) [reply src/routes/Review/index.jsx]: Done

**Review by testflyjets — APPROVED** [review]

**Pull request merged**